File: Player.py
import Transferable

class Player(Transferable.Transferable):
    def __init__(self, name: str, birth_year: int, position: str):
        super().__init__()
        self.name = name
        self.birth_year = birth_year
        self.position = position

    def __str__(self):
        return self.name + ", " + self.position

    def __lt__(self, other):
        if self.name < other.name:
            return True
        if self.position < other.position:
            return True
        if self.birth_year < other.birth_year:
            return True
        return False

File: Player.java
import java.util.HashSet;

public class Player extends Transfarable{
  private String name, position;
  private int birth_year;

  public Player(String name, int birth_year, String position){
    this.name = name; this.position = position;
    this.birth_year = birth_year;
    this.transfers = new HashSet<Transfer>();
  }

  public String toString(){
    String string = name + ", " + position;
    return string;
  }
}
